ProcessOptimizer/model_systems/color_pH.py

```diff
@@ -0,0 +1,339 @@
+import csv
+import math
+import numpy as np
+import os
+
+from typing import List
+from .model_system import ModelSystem
+from ..space import Integer
```

Comment:
```suggestion
from ProcessOptimizer.space import Integer
```

Same as above

Also with importing Integer into `ProcessOptimizer/__init__.py`